openreview/conference/templates/bidWebfield.js

```diff
@@ -345,5 +250,127 @@ function addEdgesToNotes(validNotes, edgesMap) {
   return validNotes;
 }
 
+function updateNotes(notes) {
```

Comment:
I fixed the issue with the tag widget events, I'm calling to re render the whole tab panel when the sorting criteria changes, it should be working now.